### ISSUE TITLE
White screen - Rev-C compatibility?

### ISSUE THREAD

**Issue by can-caglar** [OLIMEX/ESP32-ADF#4]
Is the SIP VoIP example compatible with the Rev C board?

I am using an ESP32-ADF rev C with a MOD-LCD2.8RTP board. When flashing the screen remains white and no particular audio plays on the audio.

**Comment by DanKoloff** [comment]
Yes, it was updated two months ago. Did you try different audio type? Did you try different options in make menuconfig?

Check this commit that added revision C specifics:

https://github.com/OLIMEX/sip_phone_example/commit/5fc350a07d17ebc43190c44a35fc35ec5114744d

**Comment by can-caglar** [comment]
I did try different audio types. The one thing I haven't done yet is set up the SIP URI information. Is that necessary before the screen is booted?
